### PR TITLE
Functional admin-side user resume table

### PR DESCRIPTION
**Pull request by trevorkw7**
Draft request. Admin-side user resume table is functional but needs some type definition and css cleanup work.

### CODE REVIEW

File: src/admin/components/UserResumeTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        if (r.status === 200) return r.blob();
+        return Promise.reject(new Error(r.statusText));
+      });
+      const name = decodeURI(url.substring(url.lastIndexOf('/') + 1));
+      folder?.file(name, promise);
+    });
+    zip.generateAsync({ type: 'blob' }).then((blob) => saveAs(blob, filename));
+  };
+
+  const handleDonwloadZip = () => {
```
Comment (raymosun) [inline src/admin/components/UserResumeTable/index.tsx:51]: Looks like download is misspelled here

File: src/admin/components/UserResumeTable/index.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      <div className="download-button-container">
+        {selectedRowKeys.length > 0 ? (
+          <Button type="primary" onClick={handleDonwloadZip} className="download-button">
+            Download ZIP
+          </Button>
+        ) : (
+          <Button type="primary" className="download-button-disabled" disabled>
+            Download ZIP
+          </Button>
+        )}
```
Comment (raymosun) [inline src/admin/components/UserResumeTable/index.tsx:101]: To achieve this, you could have one button with `disabled={selectedRowKeys.length < 1}` (stole this idea from store code 😉 )

**Review by raymosun — CHANGES_REQUESTED** [review]
Looks really good! Just noticed some minor things.

**Review by farisashai — APPROVED** [review]
Looks good to merge once we fix Raymond's comments 🚀

**Review by raymosun — APPROVED** [review]
Noice

**Pull request merged**